docs/matrix-os/examples/dash-create.md

```diff
@@ -71,14 +71,14 @@ widgets:
     label: from device
 
 screens:
-  - message
```

Comment:
this is intentional

Reply:
Sean, everytime I tried without an additional dash the matrix OS said there was a formatting error. Should we keep only one dash anyway?

Reply:
Something happened. it used to be two dashes. :(